ministryofjustice/hmpps-resettlement-passport-ui
# RP2-813 and RP2-815: Appointments form, confirm answers, validate

**Features**

- Form elements (need confirmation for content and error messaging)
- Populate input fields with pre-existing data when returning from check answers page or when failed validation
- Validation for required fields
- Validation for future date
- Persist data to check answers page
- Conditionally render table rows for the optional data (location and notes)
- Dynamic links back to appointment form elements
- Submit data to POST appointment and redirect user to the overview page appointments section
- Add BENEFITS to appointment type and to constants/enums
